### PR TITLE
vdk-jupyter: add download-key to the VDK menu

### PR DESCRIPTION
**Pull request by yonitoo**
What:
Add a new option in the VDK dropdown menu that enables you to download keytab of a Cloud-deployed data job.

Why: 
Because this functionality is present in the CLI but not in the Jupyter UI.

Testing Done: 
Added tests and all, including the existing ones, pass.

VDK Menu

Download Key Dialog

Success message

Signed-off-by: Yoan Salambashev <ysalambashev@vmware.com>

### CODE REVIEW

**Comment by duyguHsnHsn** [comment]
Can you add the result dialog of the operation, as well

**Review by duyguHsnHsn — APPROVED** [review]
Seems good to me!

File: projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/src/components/DownloadKey.tsx [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+          label="Job Name:"
+        ></VDKTextInput>
+        <VDKTextInput
+          option={VdkOption.TEAM}
+          value="default-team"
+          label="Job Team:"
+        ></VDKTextInput>
+        <VDKTextInput
+          option={VdkOption.PATH}
+          value={this.props.jobPath}
```
Comment (antoniivanov) [inline projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/src/components/DownloadKey.tsx:40]: The key should not be downloaded inside the job directory but alongside it. 

```
/root
|-- job-dir/
|   |-- config.ini
|   |-- ...
|-- job-key

 ```

Comment (yonitoo) [reply projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/src/components/DownloadKey.tsx:40]: changed this to the parent directory

**Comment by murphp15** [comment]
@yonitoo can you update the picture for the download key dialog now that the folder is changed.

File: projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/src/components/DownloadKey.tsx (hunk 1 above)
Comment (murphp15) [reply projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/src/components/DownloadKey.tsx:40]: @antoniivanov  if they save it somewhere else will it use it? 
Or does it need to be saved exactly there? 
maybe they should not be able to choose where to save it to. 
Make it simpler for customers.

Comment (antoniivanov) [reply projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/src/components/DownloadKey.tsx:40]: If they save it somewhere else , they could configure it (using [config described here](https://github.com/vmware/versatile-data-kit/tree/main/projects/vdk-plugins/vdk-kerberos-auth#configuration) which nobody ever does . By large vdk download-key CLI downloads it in the right location . If for some reason it is not they would move it.

**Comment by yonitoo** [comment]
Updated the screenshot!

**Comment by murphp15** [comment]
IMO, If we take the approach of building a press release first and then build the product after this dialog should 100% not be present!

**Pull request closed without merging**